draft-ietf-quic-transport.md

```diff
@@ -2966,19 +2968,21 @@ connection and stream flow-control windows, even though these frames will be
 discarded upon receipt.  This avoids potential ambiguity about which STREAM
 frames count toward flow control.
 
-STOP_SENDING can only be sent for any stream that is not "idle", however it is
-mostly useful for streams in the "open" or "half-closed (local)" states.  A
-STOP_SENDING frame requests that the receiving endpoint send a RST_STREAM frame.
-An endpoint that receives a STOP_SENDING frame MUST send a RST_STREAM frame for
-that stream with an error code of STOPPING.  If the STOP_SENDING frame is
-received on a stream that is already in the "half-closed (local)" or "closed"
-states, a RST_STREAM frame MAY still be sent in order to cancel retransmission
-of previously-sent STREAM frames.
+A STOP_SENDING frame requests that the receiving endpoint send a RST_STREAM
+frame.  An endpoint that receives a STOP_SENDING frame MUST send a RST_STREAM
+frame for that stream, and can use an error code of STOPPING.  If the
+STOP_SENDING frame is received on a send stream that is already in the "Data
+Sent" state, a RST_STREAM frame MAY still be sent in order to cancel
+retransmission of previously-sent STREAM frames.
 
-While STOP_SENDING frames are retransmittable, an implementation MAY choose not
-to retransmit a lost STOP_SENDING frame if the stream has already been closed
-in the appropriate direction since the frame was first generated.
-See {{packetization}}.
+STOP_SENDING can only be sent for a receive stream that has not been
+reset. STOP_SENDING is most useful for streams in the "Recv" or "Size Known"
+states.
+
+An endpoint is expected to send STOP_SENDING frames if packets containing the
+frame are lost.  However, once either all stream data has been received or a
+RST_STREAM frame - that is, any receive stream state other than "Recv" or "Size
+Known" - sending the frame is not necessary.
```

Comment:
Can this be formulated more clearly? I think the paragraph is discussing retransmission while avoiding using the loaded term. E.g. what does "the frame is lost" mean? Which frame is lost? Does it mean something like: A STOP_SENDING frame should be retransmitted in case of packet loss until acknowledged or a FIN or RST_STREAM signal has been received.